src/Function.cpp

```diff
@@ -897,6 +932,13 @@ OutputForwardDecl(Function *func, std::ostream *pOut)
 	return 0;
 }
 
+static int
+OutputForwardDeclAlias(Function *func, std::ostream *pOut)
```

Comment:
Why do we need this function? Where is it used?

Reply:
It is used to provide alias function name declaration. 

For example - In current state of csmith, function declarations are output as - 

void func_1();
void func_5();
...so on

Since, we are creating aliases for functions, alias names are fuzzed randomnly thorughout the program. Hence, for above function declarations, I am fuzzing alias function declaration as - 

void func_1_alias();
void func_5_alias();
...so on